.github/workflows/build.yml

```diff
@@ -0,0 +1,67 @@
+name: "Build"
+on: push
+env:
+  PREFIX: /opt/tenzir
+  CC: cc
+  CXX: c++
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
+        configure: ["", "--dev-mode"]
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: recursive
+          ref: ${{ github.ref }}
+      - uses: actions/setup-python@v1
+        with:
+          python-version: "3.8"
+      - name: Install Ubuntu dependencies
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
Is there a reason for switching away from debian?

Reply:
There is no Debian in github actions.

Reply:
Ah, I see.